lib/customer.rb

```diff
@@ -0,0 +1,39 @@
+require 'csv'
+# require 'awesome_print'
+
+class Customer
+  attr_reader :id
+  attr_accessor :email, :address
```

Comment:
These should be `attr_reader` methods instead of `attr_accessor`.  We don't need to give users of the class the ability to directly change the instance variables.  Reading them is enough.  In general we give the minimal access we can to accomplish the task.  This prevents the users of our class from assigning email and address to invalid values.